src/app/modals/property-management-modal/property-management-modal.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+import { propertyForm } from '../../../form/form.service';
+import { NgbModal, NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+
+@Component({
+  selector: 'app-property-management-modal',
+  templateUrl: './property-management-modal.component.html',
+  styleUrl: './property-management-modal.component.css'
+})
+export class PropertyManagementModalComponent implements OnInit {
+  form = propertyForm
+  constructor(public activeModal: NgbActiveModal) {
+    
+  }
+  ngOnInit(): void {
+    console.log(this.form.controls.title)
```

Comment:
i believe we don't want this console.log

Reply:
Which means you'll have to remove the OnInit implementation as well because it wont be used, at least for now.